src/model.py

```diff
@@ -14,8 +14,21 @@ def __init__(self, forward_only, batch_size, use_lstm=True):
         self.batch_size = batch_size
         single_cell = tf.contrib.rnn.GRUCell(config.EMBEDDING_SIZE)
         if use_lstm:
-          single_cell = tf.nn.rnn_cell.BasicLSTMCell(config.EMBEDDING_SIZE)
-        self.cell = tf.contrib.rnn.MultiRNNCell([single_cell for _ in range(config.NUM_LAYERS)])
+          single_cell = tf.nn.rnn_cell.LSTMCell(config.EMBEDDING_SIZE)
+        """
+        TODO:
+        Add dropout layer here, maybe?
+        """
+        input_keep_prob=1.0
+        output_keep_prob=1.0
+        state_keep_prob=1.0
+        drop_out_cell = tf.nn.rnn_cell.DropoutWrapper(single_cell, input_keep_prob, output_keep_prob, state_keep_prob)
```

Comment:
[Reference: DropoutWrapper](https://www.tensorflow.org/api_docs/python/tf/nn/rnn_cell/DropoutWrapper)

Reply:
For the dropout layer part, we need to add the DropoutWrapper behind each LSTMCell or GRUCell.
To implement it, you can refer to this part of code:
```
self.keep_drop = tf.placeholder(tf.float32)
def create_rnn_cell():
   encoDecoCell = tf.contrib.rnn.BasicLSTMCell(hidden_size)
   encoDecoCell = tf.contrib.rnn.DropoutWrapper(encoDecoCell, input_keep_prob=1.0, output_keep_prob=self.keep_drop)
return encoDecoCell
encoCell = tf.contrib.rnn.MultiRNNCell([create_rnn_cell() for _ in range(num_layers)])
```
Reference: [seq2seq_model](https://github.com/lc222/seq2seq_chatbot/blob/master/seq2seq_model.py)

Reply:
Sorry this snippet of code is outdated, the dropout layer is added correctly in the latest code.

Reply:
OK, I see the correct code from `View changes`, but the code in `master/src/model.py` is still the outdated one. What should I do to get the correct one?